### ISSUE TITLE
Event listeners are not triggered when app is running in the background

### ISSUE THREAD

**Issue by MehediH** [cjam/react-native-spotify-remote#227]
Any of the event listeners added to SpotifyRemote don't get triggered when the app is running in the background. It only works when the app is active and in focus - is this intentional? 

Is there anyway to have the event listeners work even when the app is running in the background (including when the phone is locked)? Mainly need this for the `playerStateChanged` event. Thank you!

**Comment by cjam** [comment]
hmm.. You're talking about within your app?  I had a similar issue and it was due to iOS backgrounding / sleeping my app and disconnecting the remote connection with spotify.  You just need to employ a strategy to keep your app from being backgrounded.  There are a few sdk's that if in use will prevent iOS from backgrounding your app.  Audio and GPS are two that come to mind that you could investigate. 

It's pretty lame, I wish there was some sort of real time entitlement for iOS that you could use to prevent your app from being backgrounded.  Maybe there is?  worth an investigation.

**Comment by MehediH** [comment]
Yes within my app. Do you have an example of how you got it to stop iOS from backgrounding the app?

**Comment by Pashkagreen** [comment]
> Yes within my app. Do you have an example of how you got it to stop iOS from backgrounding the app?

Hello, @MehediH! Have you solved this problem?
